jobs/demo-district-update-pass.groovy

```diff
@@ -0,0 +1,85 @@
+// Automate monthly change for PL sandbox account passwords
+// https://khanacademy.atlassian.net/browse/DIST-4092
+
+@Library("kautils")
+// Classes we use, under jenkins-jobs/src/.
+import org.khanacademy.Setup;
+// Vars we use, under jenkins-jobs/vars/.  This is just for documentation.
+//import vars.exec
+//import vars.notify
+//import vars.withTimeout
+//import vars.withSecrets
+
+new Setup(steps
+
+).allowConcurrentBuilds(
+
+).addStringParam(
+   "CYPRESS_GIT_REVISION",
+   """The name of a cypress branch to use when building.""",
+   "update-psw"
+
+).addStringParam(
+   "SLACK_CHANNEL",
+   "The slack channel to which to send failure alerts.",
+   "#demo-district-logs"
+
+).addStringParam(
+   "REVISION_DESCRIPTION",
+   """Set by the buildmaster to give a more human-readable description
+of the GIT_REVISION, especially if it is a commit rather than a branch.
+Defaults to GIT_REVISION.""",
+   "master"
+
+).addStringParam(
+   "TEST_RETRIES",
+   """How many retry attempts to use. By default is 1.""",
+   "1"
+
+).apply()
+
+// We use the build name as a unique identifier for user notifications. 
+BUILD_NAME = "build demo-district-password-update #${env.BUILD_NUMBER} (${params.REVISION_DESCRIPTION})"
```

Comment:
This is overkill for this job -- only one of these ever runs at a time.  You can look at other similar periodic jobs, like update-devserver-static-images.groovy, to see what kind of setup they do.

(I'd get rid of REVISION_DESCRIPTION entirely as well.)